VAMobile/src/api/appointments/cancelAppointment.tsx

```diff
@@ -65,10 +64,7 @@ export const useCancelAppointment = (currentPage: number) => {
         },
       }
 
-      queryClient.setQueryData(
-        [appointmentsKeys.appointments, TimeFrameTypeConstants.UPCOMING, currentPage],
-        newAppointments,
-      )
+      queryClient.setQueryData([appointmentsKeys.appointments, TimeFrameTypeConstants.UPCOMING], newAppointments)
```

Comment:
This was the lone remaining api/ example that included 'currentPage' so I think we are safe saying this is the only lingering example.